### ISSUE TITLE
plotting resids with PDC

### ISSUE THREAD

**Issue by mnfienen** [pypest/pyemu#164]
In cases where `++ies_drop_conflicts` is True, it would be nice to eliminate those dropped obs (especially if they are really really bad!) from plots like 1to1 plots. A simple flag could be supplied to `pst.plot(kind='1to1', drop_pdc=True)` for example, that reads the csv file with PDC-dropped obs and just removes them before plotting.

I can implement, but wanted to see if anyone has objections or other ideas of how to deal.

**Comment by jwhite-usgs** [comment]
I think we do need to get better at handling conflicts in general.  But instead of dropping, what about having them washed out or otherwise colored differently or at least have that option?  There is a more advanced PDC detection routine in helpers now so we could potentially just call that under the hood in the plotting so that we dont have an external file dependency (that may change in the future!).  Just my thoughts.

**Comment by mnfienen** [comment]
Yeah - I do think this might be useful (although I've been going rogue with 1to1 plots lately). Maybe could have two options for `pdc_plot` like 'drop' or 'gray'? If that's cool, maybe I can implement.

Also, for some recent work, found it valuable to plot 1to1 with error bars in both obs and modeled like the attached. Maybe can fold that into pyemu?

![image](https://user-images.githubusercontent.com/1110827/120000049-05e8a900-bf98-11eb-9b3d-7d5ba077b85b.png)

**Comment by jtwhite79** [comment]
Those plots look awesome!  We could also just have variants on the 1-to-1 plots since all of us GW modelers are so obsessed with it - like a `Pst.plot(kind="1to1-fancysauce")` ?  I like showing the meas error bars too!

**Comment by mnfienen** [comment]
@jtwhite79  nope lol. I can try and revisit though!

**Comment by briochh** [comment]
@mnfienen, happy for me to move this to discussions? Might be good to revisit when whoever has the time?
